cloudhub-2/modules/ROOT/pages/ps-config-domains.adoc

```diff
@@ -89,9 +89,7 @@ To enable traffic to reach the apps deployed to the private space using a custom
 .. Log in to your domain registrar.
 +
 Use https://www.whois.com/[Whois^] to look up your registrar.
-.. Click *Copy* to copy the DNS target for the current private space.
-// <<screenshot>>
-.. Replace the domain's nameserver by pasting the DNS target.
+.. Create a CNAME record in your vanity domain, pointing to your private space’s FQDN.
```

Comment:
According to our MuleSoft CX Writing Style Reference, we should not use the possessive case except for Salesforce, MuleSoft, and people. In this case "space FQDN" would be fine.

Reply:
Applied